src/services/migrations.ts

```diff
@@ -3,16 +3,27 @@ import { getLineFiltersVariable } from './variableGetters';
 import { LineFilterCaseSensitive } from '../Components/ServiceScene/LineFilter/LineFilterScene';
 import { LineFilterOp } from './filterTypes';
 import { ServiceScene } from '../Components/ServiceScene/ServiceScene';
+import { urlUtil } from '@grafana/data';
+
+function removeEscapeChar(value: string) {
+  return value
```

Comment:
Curious about the unusual approach. Why not a regex?

Reply:
Was struggling to get the regex to work last night, I'll take another look

Reply:
Nevermind, I made this implementation more complicated to fix the user-entered escape char case, so I think it's beyond my regex abilities without dumping a bunch more time into this.